azext_iot/central/_help.py

```diff
@@ -1375,6 +1650,232 @@ def _load_central_edge_help():
+            --device-id {deviceId}
+            --component-name {componentName}
+            --module-name {moduleName}
+    """
+
+    helps[
+        "iot central device edge module properties update-component-properties"
+    ] = """
+        type: command
+        short-summary: Update module properties for a specific component via patch.
+
+        examples:
+        - name: Update module properties for a specific component via patch.
+          text: >
+            az iot central device edge module properties update-component-properties
+            --app-id {appid}
+            --device-id {deviceId}
+            --component-name {componentName}
+            --module-name {moduleName}
+            --content {json}
+    """
+
+    helps[
+        "iot central device edge module properties replace-component-properties"
+    ] = """
+        type: command
+        short-summary: Replace module properties for a specific component.
+
+        examples:
+        - name: Replace module properties for a specific component.
+          text: >
+            az iot central device edge module properties replace-component-properties
+            --app-id {appid}
+            --device-id {deviceId}
+            --component-name {componentName}
+            --module-name {moduleName}
+            --content {json}
+    """
+
+    helps[
+        "iot central device edge module show-component-telemetry-value"
+    ] = """
+        type: command
+        short-summary: Get the last telemetry value from a module component.
+        examples:
+        - name: Get module component telemetry value.
+          text: >
+            az iot central device edge module show-component-telemetry-value
+            --app-id {appid}
+            --device-id {deviceId}
+            --component-name {componentName}
+            --module-name {moduleName}
+            --telemetry-name {telemetryName}
+    """
+
+    helps[
+        "iot central device edge module properties show"
+    ] = """
+        type: command
+        short-summary: Get all property values of a module.
+        examples:
+        - name: Get module properties.
+          text: >
+            az iot central device edge module properties show
+            --app-id {appid}
+            --device-id {deviceId}
+            --module-name {moduleName}
+    """
+
+    helps[
+        "iot central device edge module properties update"
+    ] = """
+        type: command
+        short-summary: Update property values of a module.
+        examples:
+        - name: Update module properties via patch.
+          text: >
+            az iot central device edge module properties update
+            --app-id {appid}
+            --device-id {deviceId}
+            --module-name {moduleName}
+            --content {json}
+    """
+
+    helps[
+        "iot central device edge module properties replace"
+    ] = """
+        type: command
+        short-summary: Replace all property values of a module.
```

Comment:
will replace overwrite while update merges the changes with the original properties?

Reply:
We'll overwrite all writable properties through replace API.